src/objc.h

```diff
@@ -0,0 +1,75 @@
+
+/* Compiler implementation of the D programming language
+ * Copyright (c) 2015 by Digital Mars
+ * All Rights Reserved
+ * written by Michel Fortin
+ * http://www.digitalmars.com
+ * Distributed under the Boost Software License, Version 1.0.
+ * http://www.boost.org/LICENSE_1_0.txt
+ * https://github.com/D-Programming-Language/dmd/blob/master/src/objc.h
+ */
+
+#ifndef DMD_OBJC_H
+#define DMD_OBJC_H
+
+#include "root.h"
+#include "stringtable.h"
+
+class Identifier;
+class FuncDeclaration;
+class ClassDeclaration;
+class InterfaceDeclaration;
+class ObjcSelector;
+
+class ObjcSelector
+{
+public:
+    static StringTable stringtable;
+    static StringTable vTableDispatchSelectors;
+    static int incnum;
+
+    const char *stringvalue;
+    size_t stringlen;
+    size_t paramCount;
+
+    static void init ();
```

Comment:
Got a GNU-style convention going on here (space between method name and parameters)